frontend/src/main/webapp/WEB-INF/config.sample.properties

```diff
@@ -114,18 +114,25 @@ admin.permission=administrator
 #CAS logout URL
 #cas.logout.url=https://192.168.1.8:8443/cas2/logout?service=http%3A%2F%2F192.168.7.42%3A18080%2Funifiedviews%2F
 
-#type of ownership user|organization
-ownership.type=user
+#CAS attribute representing full name
+cas.subjectIdAttributeName = Subject.UPVSIdentityID
+
+#CAS attribute representing full name
+cas.fullNameAttributeName = Subject.FormattedName
```

Comment:
proposed change of name to cas.attributeName.fullName
also relevant for others to improve readability